diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/StringUtils.kt

```diff
@@ -63,3 +64,32 @@ fun String.removePrefix(): String {
     if (this.isUpperSnakeCase()) return this.substring(2)
     return this
 }
+
+fun String.editorDistance(another: String): Int {
+    val acc = Array(length) {
+        Array(another.length) { 0 }
+    }
+
+    acc[0].indices.forEach {
+        acc[0][it] = it
+    }
+
+    acc.indices.forEach {
+        acc[it][0] = it
+    }
+
+    (1 until acc.size).forEach { row ->
+        (1 until acc[0].size).forEach { col ->
```

Comment:
why acc[0] is here and not row?

Reply:
Because all rows have the same size. Could've used string length here, though.